test/tattr.c

```diff
@@ -6978,11 +7286,22 @@ attr_iterate_check(hid_t fid, const char *dsetname, hid_t obj_id, H5_index_t idx
     unsigned oskip;   /* # of attributes to skip on object, with H5Aiterate1 */
 #endif                /* H5_NO_DEPRECATED_SYMBOLS */
     int    old_nerrs; /* Number of errors when entering this check */
-    herr_t ret;       /* Generic return value */
+    bool   vol_is_native;
+    herr_t ret; /* Generic return value */
 
     /* Retrieve the current # of reported errors */
     old_nerrs = GetTestNumErrs();
 
+    if (!(vol_cap_flags_g & H5VL_CAP_FLAG_ITERATE)) {
+        SKIPPED();
```

Comment:
There are a few instances of `SKIPPED()` - should these be `MESSAGE(5, (" -- SKIPPED --\n"));` for consistency?

Reply:
In general I tried to use MESSAGE only if there is any output at the start of a test function which is also printed out with MESSAGE. That way the SKIPPED message is only printed if output for the test is printed. In this case, it probably even makes more sense to just return without printing skipped since there isn't any output nearby. I don't believe these capability flag changes were ever checked with a VOL connector that would fail at this point in order to see what the output looks like. That's a good todo item.